### ISSUE TITLE
About hyperparameters

### ISSUE THREAD

**Issue by alen0216056** [cvlab-epfl/segmentation-driven-pose#2]
Hello, 
I am trying to implement this work according to the paper, but I don't know the value of some hyperparameters. 
Can you provide the value of 'tau' used in the confidence loss and the value of 'beta' and 'gamma' used in the regression loss?
Thank you very much :)

**Comment by YinlinHu** [comment]
Great, Thanks for your interests in our work. if the total loss expressed as L=aL(class)+bL(position)+cL(confidence), the a,b,c have values close to 0.05, 0.01, 0.001. While we did not put much efforts into fine-tuning the network. Maybe you can find more suitable values.

**Comment by sjtuytc** [comment]
we use the same loss hyperparameters and find that the position loss and confidence loss don't appear to reduce.

**Comment by alen0216056** [comment]
In my implementation, I used 50, 10, 1 as the loss hyperparameters respectively.
The ratio between my loss hyperparameters is as same as those provided by the author.
I think the exact value of hyperparameters is depend on your implementation, you can observe the trend of the loss curve and try to set a proper value.
